tests/verilog/bug2042.ys

```diff
@@ -0,0 +1,12 @@
+logger -expect error "Non-ANSI style task/function arguments not currently supported" 1
+read_verilog <<EOT
+module Task_Test_Top
+(
+);
+
+    task SomeTaskName(a)
```

Comment:
I would suggest fixing the other syntax error (missing `;` at the end of the line) in this test, in case we change the frontend in a way that finds the other error first at some point...